CHANGELOG.md

```diff
@@ -7,6 +7,37 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.9.0] - December, 21, 2022
+
+### Design kit
+
+#### Added
+
+- [Tag] Possibility to override colours when using Tag symbols in Sketch
+
+### React Components
+
+#### Added
+
+- [Accordion] Possibility to set notification heading level
+
+#### Fixed
+
+- [DateInput] Datepicker performance improvements
+- [DateInput] Unnecessary error logs removed from the tests
+
+### Core
```

Comment:
Nothing changed under Core package, I think we can remove this mention.

Reply:
- [x] remove Core section